docs/sdk/serverless-signing/eip191.md

```diff
@@ -1,58 +1,48 @@
 import FeedbackComponent from "@site/src/pages/feedback.md";
 
-# Signing with EIP191
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-Use EIP191 `eth_personal_sign` to sign a message instead of a transaction or raw signature.
+# Signing within a Lit Action using EIP-191
 
-This can be done with
+## Overview
 
-```js
-LitActions.ethPersonalSignMessageEcdsa({ message, publicKey , sigName }); 
-```
+Lit Actions offer the ability to use [EIP-191](https://eips.ethereum.org/EIPS/eip-191?ref=blog.spruceid.com) to sign a message instead of a transaction or raw signature. This is done with the Lit Actions method [ethPersonalSignMessageEcdsa](https://actions-docs.litprotocol.com/#ethpersonalsignmessageecdsa). 
 
-It will prepend `"\x19Ethereum Signed Message:\n"` to the message and then hash and sign it according to https://eips.ethereum.org/EIPS/eip-191.
+It will prepend `"\x19Ethereum Signed Message:\n"` to the message and then hash and sign it according to the EIP-191 standard.
```

Comment:
```suggestion
It will prepend to your message: `"\x19Ethereum Signed Message:\n"` and the length of the message, then hash and sign it according to the EIP-191 standard.
```